importer/tests/test_views.py

```diff
@@ -52,9 +52,8 @@ def test_upload_duplicate(self):
         self.assertTrue(Course.objects.count() == 0)
         self.upload_test_file()
         self.assertTrue(Course.objects.count() == 1)
-        body = self.upload_test_file()
+        self.upload_test_file()
         self.assertTrue(Course.objects.count() == 1)
```

Comment:
I'll go ahead and remove that exception handler as well now too, but it might be worthing talking more about this if you'd like.  It certainly isn't better for the UX until that code is completed